### ISSUE TITLE
"Circular Render Distance" Toggle Option

### ISSUE THREAD

**Issue by solonovamax** [FlashyReese/sodium-extra-fabric#41]
A toggle for a "circular render distance" is something that might be really nice. Because, this is a feature that I, and many others, might find useful.

Since a visual explanation is probably best, here's what would be rendered for a render distance of 6 chunks:

![image](https://user-images.githubusercontent.com/46940694/124668671-310bc580-de7f-11eb-8d78-3f04712ec7cb.png)
### Key:
- Red Dot: Where the player is standing.
- Red Circle: A circle with a radius of 6 chunks.
- Green Line: a 6 chunk long line.
- Blue Square: Chunks that are rendered.
- Yellow Square: Chunk that *would have* been rendered, according to vanilla behaviour, but are *not* rendered with "circular render distance".
- White Square: Chunk that isn't rendered.

Basically, chunks would be rendered in a circle instead of in a square around the player. This would allow players to increase their render distance slightly, because it wouldn't render the corner chunks which are further from the player.

**Comment by FlashyReese** [comment]
> Yellow Square: Chunk that would have been rendered, according to vanilla behaviour, but are not rendered with "circular render distance".

This is cull by Sodium, you can turn off Fog Occlusion Culling in the advanced tab.

**Comment by solonovamax** [comment]
Ah. Sick.
